lib/ansible/modules/network/ios/ios_vlan.py

```diff
@@ -181,7 +181,7 @@ def map_obj_to_commands(updates, module):
         for h in have:
             obj_in_want = search_obj_in_list(h['vlan_id'], want)
             if not obj_in_want and h['vlan_id'] != '1':
-                commands.append('no vlan {0}'.format(h['vlan_id']))
+                commands.append('no vlan {}'.format(h['vlan_id']))
```

Comment:
No, it's not that. If was asking you to remove this line: https://github.com/pierremahot/ansible/blob/6c12bc81cf45fab9f3f9940bf6b3cd73bb1396d8/test/sanity/pylint/ignore.txt#L50